examples/configs/nemotron/config.yml

```diff
@@ -0,0 +1,6 @@
+models:
+  - type: main
+    engine: nim
+    model: nvidia/llama-3.1-nemotron-ultra-253b-v1
+    reasoning_config:
+      remove_reasoning_traces: False # set it true if you like
```

Comment:
Comment seems redundant. Perhaps "Set True to remove traces from the internal tasks" or whatever stage it's being removed from